src/components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import "./Form.css";
+
+export default function Form({onAddActivity}) {
+    function handleSubmit(event) {
+        event.preventDefault();
+        const formData = new FormData(event.target); 
+        const data = Object.fromEntries(formData);
+        const form = event.target
+      
+        const isForGoodWeather = data.isForGoodWeather === "on"? true : false;
```

Comment:
the result of the comparison `data.isForGoodWeather === "on"` is always true or false, which means you could write this even shorter:

```js
const isForGoodWeather = data.isForGoodWeather === "on";
```